src/options.ts

```diff
@@ -82,17 +82,22 @@ export const getSourcemaps = (): string[] => {
 };
 
 /**
- * Find out from input if we should finalize the release.
+ * Find out from input if we should perform an action.
```

Comment:
What do you think about this instead?

```suggestion
 * Fetch boolean option from input. Throws error if option value is not a boolean.
```